modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -81,10 +81,15 @@ echo "${forseti_environment}" > /etc/profile.d/forseti_environment.sh | sudo sh
 gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
 gsutil cp -r gs://${storage_bucket_name}/rules ${forseti_home}/
 
+# Download the Newest Config Validator constraints from GCS
+rm -rf /home/ubuntu/policy-library
```

Comment:
We should prefer `$USER_HOME` instead of the hardcoded `/home/ubuntu` path for consistency.

Reply:
updated.